src/com/company/Main.java

```diff
@@ -1,75 +1,81 @@
 package com.company;
 
 import java.io.File;
-import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Paths;
-import java.util.StringTokenizer;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import com.dbconnectors.DbConnector;
 
 public class Main {
-    final static String FILE_TYPE_TXT = ".txt";
-    final static String FILE_TYPE_JSON = ".json";
-    final static String SPECIAL_CHAR_REMOVAL_REGEX = "[^a-zA-Z0-9@]";
-    final static String SINGLE_SPACE = " ";
+    static int count = 0;
 
     public static void main(String[] args) {
-
         if (args.length != 2) {
-            System.out.println("Exiting, since no file path and word to search for are provided!");
+            System.out.println(Constants.errorMessage1);
+            DbConnector database = new DbConnector(Constants.txtFilePath, Constants.searchWord, count, Constants.status, Constants.errorMessage1);
+            database.start();
+            return;
+        }
+        Constants.txtFilePath = args[0];
+        Constants.searchWord = args[1];
+        if (Constants.searchWord == null || Constants.searchWord.equals(" ") || Constants.searchWord.equals("")) {
```

Comment:
What if the searchWord is just three empty spaces, or five empty spaces or more?
Replace with 
```suggestion
        if (Constants.searchWord == null || Constants.searchWord.trim().isEmpty()) {
```

Reply:
replace with the code that you have sent